ijwb/ijwb.bazelproject

```diff
@@ -14,11 +14,13 @@ targets:
 workspace_type: intellij_plugin
 
 build_flags:
-  --define=ij_product=intellij-latest
+  --define=ij_product=intellij-2024.3
```

Comment:
Maybe intllij-oss-under-dev?

Reply:
Because, sooner or later this will be outdated.

Reply:
`intellij-latest` is outdated by itself and using some dated version which is far from being "latest"

Reply:
Sure, I don't mind the change. If you think it makes it easier for external contributors go ahead.